code/src/layouts/app-footer/AppFooter.scss

```diff
@@ -0,0 +1,64 @@
+@import "@/styles/scss/foundation/all";
+
+.footer {
+  padding: spacing(18) 0;
+  background-color: $white;
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__list {
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    gap: spacing(2);
+  }
+
+  &__list-title {
+    font-size: $fs-lg;
+    text-decoration: none;
+    margin-bottom: spacing(10);
+    font-weight: 600;
```

Comment:
Font weight should be used from design system. Maybe consider using `AppTypography`

Reply:
Fixed using `AppTypography`